doc/advopt.txt

```diff
@@ -13,7 +13,7 @@ Advanced commands:
   //genDepend               generate a DOT file containing the
                             module dependency graph
   //dump                    dump all defined conditionals and search paths
-                            see also: --dump.format:json (useful with: ` | jq`)
+                            see also: --dump.format:json (useful with: `| jq`)
```

Comment:
So it's a breaking change?

Reply:
This wasn't rendered properly before: it wasn't an inline code but a normal text containing backticks: ``(useful with: ` | jq`)``. (See the existing documentation: https://nim-lang.github.io/Nim/nimc.html#compiler-usage-command-line-switches)

Per rule 2 (line 531, https://github.com/nim-lang/Nim/blob/b552cf722ccdf7bf567aea69f3286dd53066f3b1/lib/packages/docutils/rst.nim#L531), we don't allow whitespace after the starting `` ` `` to be considered as a valid inline code block.
(And yes, keeping it as it was before would now cause a failure)